packages/teleterm/src/mainProcess/protocolHandler.ts

```diff
@@ -58,28 +58,30 @@ function urlToPath(targetUrl, { isWin }) {
 export function interceptFileProtocol({
   protocol,
   isWin,
-  userDataPath,
-  installPath,
+  installPath: installPathWithCasing,
   logger,
 }) {
   protocol.interceptFileProtocol('file', (request, callback) => {
     const target = path.normalize(urlToPath(request.url, { isWin }));
-    const realPath = fs.existsSync(target) ? fs.realpathSync(target) : target;
-    const properCasing = isWin ? realPath.toLowerCase() : realPath;
+    const realPathWithCasing = fs.existsSync(target)
+      ? fs.realpathSync(target)
+      : target;
+    // normalize casing across windows (case-insensitive) and unix (case-sensitive)
```

Comment:
Isn't macOS case-insensitive by default? Tbh it's the first time I see case normalization being done when looking at the paths. I'm not sure if we should keep it or remove it.

Reply:
I'm ok with removing it and handling later if needed to keep the code cleaner if thats ok with you